src/main/java/mesosphere/marathon/metrics/Metrics.scala

```diff
@@ -64,6 +68,11 @@ class Metrics @Inject() (val registry: MetricRegistry) {
 }
 
 object Metrics {
+  class Counter(counter: com.codahale.metrics.Counter) {
```

Comment:
You are right. For some reason, I didn't realize that it is a distinct metrics type.